extensions/service-catalog/generator/go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/fabric8io/kubernetes-client/extensions/knative/generator
 require (
 	github.com/fabric8io/kubernetes-client/generator v0.0.0
 	github.com/kubernetes-sigs/service-catalog v0.2.2
```

Comment:
This repository has been archived since May 2022.

I see API machinery version is set to 0.18.2 in [go.mod](https://github.com/kubernetes-retired/service-catalog/blob/b37daf589e7797e209028c087d072817934d5d7c/go.mod#L35)

Reply:
If the project is no longer maintained since 2022, maybe we should deprecate for 6.13.0 and remove in 7.0.0

Reply:
Shall I create an issue about this?

Reply:
I think we can deprecate it directly

Reply:
There's an issue I'm using to track all of the 7.0 changes and roadmap (I'll add an entry there)